src/procedural.php

```diff
@@ -0,0 +1,24 @@
+<?php
+namespace procedural;
```

Comment:
As this is a repo suggested by current documentation to get started, I'm not going to make it rougly inconsistent as possible while it can.